src/main/java/com/example/Library/Management/System/model/entity/Role.java

```diff
@@ -0,0 +1,8 @@
+package com.example.Library.Management.System.model.entity;
+
+public enum Role {
```

Comment:
It's not an entity, so let's create a new package & name it `enums`
```suggestion
public enum RoleEnum {
```